shell/client/admin-client.js

```diff
@@ -1182,11 +1182,24 @@ const adminRoute = RouteController.extend({
                   let looksGood;
                   if (error) {
                     looksGood = false;
+                    console.log("Sandstorm WILDCARD_HOST self-test failed. Details:");
+                    console.log(error);
+                    console.log(
+                      "Further details may be available in the JS console provided by your web " +
```

Comment:
I agree the message is confusing, since it seems to be referring to the JS console as something far away.

Perhaps say: "Error messages above this one may relate to this error."